snap/hooks/install

```diff
@@ -55,19 +55,19 @@ if [ ! -f "$SNAP_DATA/kuiper/data" ]; then
     cp "$SNAP/etc/sources/"*.yaml "$SNAP_DATA/kuiper/etc/sources"
 fi
 
-# for the kong pki setup file, we need to set the hostname as localhost
-# and then set the directory to store the cert files as $SNAP_DATA/kong/ssl
-mkdir -p "$SNAP_DATA/config/security-secrets-setup/res"
-for svc in kong vault; do 
-    CONFIG_FILE_PATH="config/security-secrets-setup/res/pkisetup-$svc.json"
-    if [ ! -f "$SNAP_DATA/$CONFIG_FILE_PATH" ]; then
-        # replace the hostname with localhost using jq
-        jq '.x509_tls_server_parameters.tls_host = "localhost"' \
-           "$SNAP/$CONFIG_FILE_PATH" > "$SNAP_DATA/$CONFIG_FILE_PATH.tmp"
-        mv "$SNAP_DATA/$CONFIG_FILE_PATH.tmp" "$SNAP_DATA/$CONFIG_FILE_PATH"
-        chmod 600 "$SNAP_DATA/$CONFIG_FILE_PATH"
-    fi
-done
+# # for the kong pki setup file, we need to set the hostname as localhost
```

Comment:
I think these lines should be removed vs. commented out, unless there's some reason you think we should preserve them here?

Reply:
yeap, i'll remove it